Tiltfile

```diff
@@ -17,10 +17,11 @@ settings = {
     "allowed_contexts": [
         "kind-cspo",
     ],
+    "local_mode": True,
     "deploy_cert_manager": True,
     "preload_images_for_kind": True,
     "kind_cluster_name": "cspo",
-    "capi_version": "v1.5.2",
+    "capi_version": "v1.6.0",
```

Comment:
```suggestion
    "capi_version": "v1.6.1",
```

Reply:
We still use `sigs.k8s.io/cluster-api v1.6.0` in go.mod